src/main/java/uk/ac/ic/wlgitbridge/git/handler/WLRepositoryResolver.java

```diff
@@ -109,7 +110,7 @@ public Repository open(
         } catch (ForbiddenException e) {
             throw new ServiceNotAuthorizedException();
         } catch (GitUserException e) {
-            throw new ServiceMayNotContinueException(e.getMessage(), e);
+            throw new ServiceMayNotContinueException(String.join("\n", e.getDescriptionLines()), e);
```

Comment:
This completely omits the message, which may be informative. Maybe just add the message on its own line before the description?